src/main/java/pages/MainPage.java

```diff
@@ -13,48 +15,48 @@ public MainPage(WebDriver driver) {
         super(driver);
     }
 
-    @FindBy(className = "search-text-input")
-    WebElement searchField;
-
-    @FindBy(className = "user-info")
+    @FindBy(xpath = ".//span[@class='user-info']")
     WebElement loginButton;
 
-    @FindBy(id = "login-form-login")
+    @FindBy(xpath = ".//input[@id='login-form-login']")
     WebElement emailLoginField;
 
-    @FindBy(id = "login-form-password")
+    @FindBy(xpath = ".//input[@id='login-form-password']")
     WebElement passwordLoginField;
 
-    @FindBy(css = "#register-form-container #register-form")
+    @FindBy(xpath = ".//button[@id='loginButton']")
+    WebElement signButton;
+
+    @FindBy(xpath = ".//form[@id='register-form']")
     WebElement registerFrame;
 
-    @FindBy(css = "#register-form-email")
+    @FindBy(xpath = ".//input[@id='register-form-email']")
     WebElement emailRegisterField;
 
-    @FindBy(css = "#register-form-password")
+    @FindBy(xpath = ".//input[@id='register-form-password']")
     WebElement passwordRegisterFieldFirst;
 
-    @FindBy(css = "#register-form-password_repeat")
+    @FindBy(xpath = ".//input[@id='register-form-password_repeat']")
     WebElement passwordRegisterFieldSecond;
 
-    @FindBy(css = ".form-btn")
-    WebElement registerSignUpButton;
-
-    @FindBy(id = "loginButton")
-    WebElement signButton;
+    @FindBy(xpath = ".//parent::form[@id='register-form']//button[@class='form-btn']")
```

Comment:
This XPath can be simplified: `.//form[@id='register-form']//button[@class='form-btn']`